packages/shared/src/types.ts

```diff
@@ -1,5 +1,5 @@
 import { MachineConfig, StateMachine } from "xstate";
-import { MachineParseResult } from "xstate-parser-demo/src/MachineParseResult";
+import { MachineParseResult } from "@xstate/machine-extractor/src/MachineParseResult";
```

Comment:
shouldn't this be imported from a built dist file? so probably like this:
```suggestion
import { MachineParseResult } from "@xstate/machine-extractor";
```